clang/include/clang/Analysis/Analyses/LifetimeSafety/LifetimeSafety.h

```diff
@@ -21,9 +21,13 @@
 #define LLVM_CLANG_ANALYSIS_ANALYSES_LIFETIMESAFETY_H
 
 #include "clang/Analysis/Analyses/LifetimeSafety/Facts.h"
+#include "clang/Analysis/Analyses/LifetimeSafety/LifetimeStats.h"
 #include "clang/Analysis/Analyses/LifetimeSafety/LiveOrigins.h"
 #include "clang/Analysis/Analyses/LifetimeSafety/LoanPropagation.h"
+#include "clang/Analysis/Analyses/LifetimeSafety/Origins.h"
 #include "clang/Analysis/AnalysisDeclContext.h"
+#include "llvm/Support/raw_ostream.h"
+#include <string>
```

Comment:
Are these somehow getting auto-added by some tool ?

string and raw_ostream are still unused.